### PR TITLE
Lambda generator

### LINKED ISSUE #1: Hello World lambda
Based on [this tutorial](https://serverless.com/framework/docs/providers/aws/examples/hello-world/node/) would be good to have an example of hello world lambda.

This example does not create an APIGW, but you can still invoke the lambda `sls invoke -f functionName`.

```json
{
  "AWSTemplateFormatVersion": "2010-09-09",
  "Description": "The AWS CloudFormation template for this Serverless application",
  "Resources": {
    "ServerlessDeploymentBucket": {
      "Type": "AWS::S3::Bucket"
    },
    "HelloLogGroup": {
      "Type": "AWS::Logs::LogGroup",
      "Properties": {
        "LogGroupName": "/aws/lambda/hello-world-dev-hello"
      }
    },
    "IamRoleLambdaExecution": {
      "Type": "AWS::IAM::Role",
      "Properties": {
        "AssumeRolePolicyDocument": {
          "Version": "2012-10-17",
          "Statement": [
            {
              "Effect": "Allow",
              "Principal": {
                "Service": [
                  "lambda.amazonaws.com"
                ]
              },
              "Action": [
                "sts:AssumeRole"
              ]
            }
          ]
        },
        "Policies": [
          {
            "PolicyName": {
              "Fn::Join": [
                "-",
                [
                  "dev",
                  "hello-world",
                  "lambda"
                ]
              ]
            },
            "PolicyDocument": {
              "Version": "2012-10-17",
              "Statement": [
                {
                  "Effect": "Allow",
                  "Action": [
                    "logs:CreateLogStream"
                  ],
                  "Resource": [
                    {
                      "Fn::Sub": "arn:aws:logs:${AWS::Region}:${AWS::AccountId}:log-group:/aws/lambda/hello-world-dev-hello:*"
                    }
                  ]
                },
                {
                  "Effect": "Allow",
                  "Action": [
                    "logs:PutLogEvents"
                  ],
                  "Resource": [
                    {
                      "Fn::Sub": "arn:aws:logs:${AWS::Region}:${AWS::AccountId}:log-group:/aws/lambda/hello-world-dev-hello:*:*"
                    }
                  ]
                }
              ]
            }
          }
        ],
        "Path": "/",
        "RoleName": {
          "Fn::Join": [
            "-",
            [
              "hello-world",
              "dev",
              "us-east-1",
              "lambdaRole"
            ]
          ]
        }
      }
    },
    "HelloLambdaFunction": {
      "Type": "AWS::Lambda::Function",
      "Properties": {
        "Code": {
          "S3Bucket": {
            "Ref": "ServerlessDeploymentBucket"
          },
          "S3Key": "serverless/hello-world/dev/1529066015893-2018-06-15T12:33:35.893Z/hello-world.zip"
        },
        "FunctionName": "hello-world-dev-hello",
        "Handler": "handler.hello",
        "MemorySize": 1024,
        "Role": {
          "Fn::GetAtt": [
            "IamRoleLambdaExecution",
            "Arn"
          ]
        },
        "Runtime": "nodejs8.10",
        "Timeout": 6
      },
      "DependsOn": [
        "HelloLogGroup",
        "IamRoleLambdaExecution"
      ]
    },
    "HelloLambdaVersionw0OHkiybILtF9P579FJku1GAWTjbFieVMuudyHfCs": {
      "Type": "AWS::Lambda::Version",
      "DeletionPolicy": "Retain",
      "Properties": {
        "FunctionName": {
          "Ref": "HelloLambdaFunction"
        },
        "CodeSha256": "PSzzisjnTvvYknuXw+QOlAvdkQZ67qXYSvgoAi9T8W0="
      }
    }
  },
  "Outputs": {
    "ServerlessDeploymentBucketName": {
      "Value": {
        "Ref": "ServerlessDeploymentBucket"
      }
    },
    "HelloLambdaFunctionQualifiedArn": {
      "Description": "Current Lambda function version",
      "Value": {
        "Ref": "HelloLambdaVersionw0OHkiybILtF9P579FJku1GAWTjbFieVMuudyHfCs"
      }
    }
  }
}
```

### PR DESCRIPTION
**Pull request by phstc**
Fix #1.

@rlister It is still missing `stax lambdas invoke lambda`, unless it is already implemented in stax (see https://github.com/woodmont/capital/pull/815/files#r199838329).

BTW I initially tried to name it `lambda` instead of `lambdas`, but it seems there's a naming conflict with stax.

```
stax lambda create
bundler: failed to load command: stax 
NoMethodError: undefined method `subcommand_help' for Stax::Lambda:Module
```

### CODE REVIEW

File: lib/generators/lambdas/templates/%stack%/lambda.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  # environment(
+  #   Variables: {
+  #     VAR_NAME: 'VALUE'
+  #   }
+  # )
+  code(
+    ZipFile: code
+  )
+  handler 'index.handler' # needed for inline code
+  runtime 'nodejs6.10'
```
Comment (phstc) [inline lib/generators/lambdas/templates/%stack%/lambda.rb:15]: @rlister just confirming: for using 8.10, we would need to zip and upload to S3, right?

Comment (phstc) [reply lib/generators/lambdas/templates/%stack%/lambda.rb:15]: nvm confirmed https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-lambda-function-code.html#cfn-lambda-function-code-zipfile

Comment (rlister) [reply lib/generators/lambdas/templates/%stack%/lambda.rb:15]: @phstc The `lambda` mixin does have an `invoke` task, but it is called `test` so you can't find it :trollface: 

https://github.com/rlister/stax/blob/master/lib/stax/mixin/lambda.rb#L84-L100

`stax $stack lambda test $id`

Comment (rlister) [reply lib/generators/lambdas/templates/%stack%/lambda.rb:15]: @phstc I think this is fine as an example, at least until I get around to fixing S3 upload support in `stax`.

Comment (phstc) [reply lib/generators/lambdas/templates/%stack%/lambda.rb:15]: @rlister sure thing `ci`.

**Pull request merged**